README.md

```diff
@@ -16,6 +16,7 @@ Spread
 [Fast iterations with reuse](#reuse)  
 [Debugging](#debugging)  
 [Repeating tasks](#repeating)
+[Define number of Workers](#workers)
```

Comment:
This (and the bits below) does not match what is now implemented. Please update or remove.

Reply:
done